client/transactions/filters/index.tsx

```diff
@@ -8,45 +8,46 @@ import { getQuery } from '@woocommerce/navigation';
 /**
  * Internal dependencies
  */
-import { filters, advancedFilters, TransactionsFilterType } from './config';
+import { getFilters, getAdvancedFilters } from './config';
 import { formatCurrencyName } from '../../utils/currency';
 import './style.scss';
+import { __ } from '@wordpress/i18n/build-types';
```

Comment:
It's not used anymore, so we can remove it.

Reply:
Good catch!